tests/C0/ProcessRunnerTest.php

```diff
@@ -2,9 +2,58 @@
 
 namespace Tests\C0;
 
+use App\ProcessRunner;
 use PHPUnit\Framework\TestCase;
 
 class ProcessRunnerTest extends TestCase
 {
-    // TODO: write 2 tests to coverage 100% code
+    private $processRunner;
+
+    public function __construct()
+    {
+        parent::__construct();
+        $this->processRunner = new ProcessRunner();   
+    }
+
+    public function testRunWhenIf1TrueAndIf2True()
```

Comment:
Nên coi class là Unit cần test, tức class là SUT (System Under Test). Vì thế test case sẽ là test hành vi,không nên quá quan tâm đến tên method là gì, bên trong nó implement như thế nào, khi đó có thể thay đổi cách implement nhưng hành vi không thay đổi nên không phải update lại unit test.

Tên method unit test không cần phải ngắn gọn, nên viết theo snake_case để dễ đọc hơn

Có 1 cách đặt tên như thế này: `Test It <expected result> When <input are ...>`

=> Test It called process1 and process3 When input1 is 12 and input2 can be whatever
```suggestion
    public function test_IT_called_process1_and_process3_WHEN_input1_is_12_input2_is_whatever()
```